### PR TITLE
Scenario, Data Acquisition, Modeling sections, minor edits

### PR DESCRIPTION
**Pull request by wdecay**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -36,19 +36,52 @@ As per the [Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/a
...
 An important design goal is modularity: the components provide utility on their own, and can be re-used in different scenarios to solve other problems at a lower incremental cost.
 
-# Scenario (business understanding)
+# Scenario
 
-TBD
+The solution deals with an imaginary IoT-enabled manufacturing environment comprised of generalized rotational equipment, which may include pumps, turbines, gearboxes, compressors, and engines. 
 
-# Data acquisition and understanding
+The machines are equipped with sensors from which telemetry is regularly transmitted to the cloud in near real time. Maintenance logs are also available and, among other things, contain records of failure events indicating exact points in time when a machine had a critical failure of a particular type.
```
Comment (miwelsh) [inline docs/README.md]: The machines are equipped with sensors that transmit telemetry to the cloud in near real time. Maintenance logs contain record the timestamp of each failure and labels of which component(s) failed.

File: docs/README.md [hunk 2]
```diff
@@ -36,19 +36,52 @@ As per the [Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/a
...
+
+The main business objective of Predictive Maintenance is preempting these events in the future by raising alarms in advance, so that appropriate preventive activities can be carried out.
```
Comment (miwelsh) [inline docs/README.md]: The objective of Predictive Maintenance is predicting failure far enough ahead of time to mitigate failures while also avoiding replacing healthy components (preventative maintenance).

File: docs/README.md [hunk 3]
```diff
@@ -36,19 +36,52 @@ As per the [Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/a
...
+
+1. Generation of static seed data
+2. Ingestion of IoT telemetry and enterprise data (maintenance logs)
+
+As shown in the diagram below, both options produce a data set of the same format. Option #1 is available as a "shortcut" for quickly (in a matter of minutes) generating an arbitrarily large input data set, whereas option #2 emulates a realistic production situation where incoming telemetry is staged over an extended period of time in storage blobs, and maintenance logs are accumulated in some other data store (exemplified as a SQL database in the diagram).
 
 ![](img/data_collection.png)
 
+The data generator included with the solution is capable of both generating static seed data (option #1) and emitting real-time telemetry (option #2).
+
```
Comment (miwelsh) [inline docs/README.md]: To better understand IoT data collection in a production space, review [the documentation for the IoT Remote Monitoring solution](https://docs.microsoft.com/en-us/azure/iot-accelerators/iot-accelerators-remote-monitoring-explore).

File: docs/README.md [hunk 4]
```diff
@@ -36,19 +36,52 @@ As per the [Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/a
...
 # Modeling
 
+Having acquired an input data set, one can proceed to the [modeling stage of TDSP](https://docs.microsoft.com/en-us/azure/machine-learning/team-data-science-process/lifecycle-modeling).
```
Comment (miwelsh) [inline docs/README.md]: After confirming that your input data collection processes are complete, and you have collected sufficient telemetry to train an accurate model, proceed to the [modeling stage of Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/azure/machine-learning/team-data-science-process/lifecycle-modeling).

TODO - How much telemetry is sufficient to train an accurate model?

File: docs/README.md [hunk 5]
```diff
@@ -36,19 +36,52 @@ As per the [Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/a
...
+
 ![](img/modeling.png)
 
-# Deployment
+Modeling is an iterative process consisting of the following procedures:
```
Comment (miwelsh) [inline docs/README.md]: Modeling is an iterative process consisting of:

File: docs/README.md [hunk 6]
```diff
@@ -36,19 +36,52 @@ As per the [Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/a
...
+
+* Feature engineering
```
Comment (miwelsh) [inline docs/README.md]: [Feature engineering](https://docs.microsoft.com/en-us/azure/machine-learning/team-data-science-process/create-features)

File: docs/README.md [hunk 7]
```diff
@@ -36,19 +36,52 @@ As per the [Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/a
...
+* Training
```
Comment (miwelsh) [inline docs/README.md]: [Model training](https://docs.microsoft.com/en-us/azure/machine-learning/studio/algorithm-choice)

File: docs/README.md [hunk 8]
```diff
@@ -36,19 +36,52 @@ As per the [Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/a
...
+* Model evaluation
```
Comment (miwelsh) [inline docs/README.md]: [Model evaluation](https://docs.microsoft.com/en-us/azure/machine-learning/studio/evaluate-model-performance)

File: docs/README.md [hunk 9]
```diff
@@ -36,19 +36,52 @@ As per the [Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/a
...
+
+Once it is confirmed (through evaluation) that the model is suitable for production, a data pipeline with scoring can be deployed to a production or production-like environment for final customer acceptance.
```
Comment (miwelsh) [inline docs/README.md]: After selecting a model for production, a data pipeline with scoring can be deployed to a production or production-like environment for final customer acceptance.

Comment (wdecay) [reply docs/README.md]: Why is this better?

File: docs/README.md (hunk 4 above)
Comment (wdecay) [reply docs/README.md]: This section is not meant ot be comprehensive. It is an intro which will reference in-depth documents. The intro wil also appear in the dashboard.

File: docs/README.md (hunk 3 above)
Comment (wdecay) [reply docs/README.md]: How is this helpful?

File: docs/README.md (hunk 1 above)
Comment (wdecay) [reply docs/README.md]: This statement doesn't match our data.

File: docs/README.md (hunk 3 above)
Comment (miwelsh) [reply docs/README.md]: We don't need to reinvent the tactics to collect real-world telemetry with IoT.  The customer may have a few questions about the data generator that we can answer, but when they are ready to use real-world data we should refer the customer to the documentation that has already been successful.

File: docs/README.md [hunk 10]
```diff
@@ -36,19 +36,52 @@ As per the [Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/a
 
 An important design goal is modularity: the components provide utility on their own, and can be re-used in different scenarios to solve other problems at a lower incremental cost.
 
-# Scenario (business understanding)
+# Scenario
 
-TBD
+The solution deals with an imaginary IoT-enabled manufacturing environment comprised of generalized rotational equipment, which may include pumps, turbines, gearboxes, compressors, and engines.
```
[the remark this answers is not in the document]
Comment (wdecay) [reply docs/README.md]: the data does not specify what type of equipment is being used.

File: docs/README.md (hunk 1 above)
Comment (wdecay) [reply docs/README.md]: The second sentence, I mean.

File: docs/README.md (hunk 3 above)
Comment (wdecay) [reply docs/README.md]: Let's add this in the productionalization document. Don't want this intro to be too heavy.

File: docs/README.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (miwelsh) [reply docs/README.md]: Right.  The data generator is simulated telemetry from "generalized rotational equipment".  What I'm trying to say is that we should explicitly say what the data generator is.  If we indicate that it has similarities to other equipment then we need to show how.  If we limit our explanation to what I recommended then the customer knows exactly what the data generator is.

File: docs/README.md (hunk 4 above)
Comment (wdecay) [reply docs/README.md]: Somehow I am really uncomfortable writing in the second person. I feel it's too direct and almost condescending. Perhaps, it's just me...

File: docs/README.md (hunk 1 above)
Comment (miwelsh) [reply docs/README.md]: Understood.  I may have fallen back to my experience from my team with AWS where all we cared about was which component needed to be replaced.  How big is the difference between "critical failure of a particular type" and "labels of which component(s) failed"?  Would the second sentence match our data better if it said "Maintenance logs contain record the timestamp of each failure and critical failure of a particular type"?  What else is encapsulated within "among other things"?

File: docs/README.md (hunk 3 above)
Comment (miwelsh) [reply docs/README.md]: I can live with that.

File: docs/README.md (hunk 7 above)
Comment (wdecay) [reply docs/README.md]: These links can be confusing because they refer to ML Studio (a different product from ML v2)

Comment (miwelsh) [reply docs/README.md]: I noticed that as well after I made the comments.  I still think we should link something, but the MLv2 doc is pretty light.

**Pull request merged**